File: PHP/8php 2/problem.php
<?php
/*
|--------------------------------------------------------------------------
| Super Duper Easy
|--------------------------------------------------------------------------
|
*/

function problem($x){
   return gettype($x) == 'string' ? 'Error' : $x * 50 + 6;
}


function problem($x){
   return is_string($x) ? 'Error' : $x * 50 + 6;
}


function problem($x){
   return preg_match("/[a-zA-Z]/", $x) ? 'Error' : ($x*50)+6;
}


print_r(problem("hello") . " Error
");
print_r(problem(1) . ' ' . 56 . '
');

File: Python/8py 3/problem.py
"""
|--------------------------------------------------------------------------
| Super Duper Easy
|--------------------------------------------------------------------------
|
"""

def problem(a):
   return 'Error' if type(a) == str else a * 50 + 6


def problem(a):
   try:
      return a * 50 + 6
   except TypeError:
      return "Error"


def problem(a):
   return "Error" if isinstance(a,str) else a*50+6


def problem(a):
   return a*50+6 if str(a).replace(".", "").isdigit() else "Error"


print(problem("hello"), "Error")
print(problem(1), 56)